packages/auth-ts/src/core/persistence/react_local.ts

```diff
@@ -0,0 +1,52 @@
+/**
+ * @license
+ * Copyright 2019 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  Persistence,
+  PersistenceType,
+  PersistenceValue,
+  Instantiator
+} from '../persistence';
+
+import AsyncStorage from '@react-native-community/async-storage';
+
+class ReactNativeLocalPersistence implements Persistence {
```

Comment:
can you add unit tests?

Reply:
I'll add in a followup PR once we establish how we would like the new SDK to be structured (this PR is finalized). I thought we were tracking these separately on the sheet so I didn't add unit tests.